packages/vats/src/lib-board.js

```diff
@@ -275,28 +271,7 @@ export const prepareBoardKit = baggage => {
        * rather than preserving object identity.
        */
       getReadonlyMarshaller() {
-        return this.facets.readonlyMarshaller;
-      },
-    },
-    /**
-     * @returns {ReturnType<import('@endo/marshal').makeMarshal>}
-     */
-    readonlyMarshaller: {
-      serialize(val) {
-        return my(this).readonlyMarshaller.serialize(val);
-      },
-      unserialize(data) {
-        return my(this).readonlyMarshaller.unserialize(data);
-      },
-    },
-    publishingMarshaller: {
-      /** @param {*} val */
-      serialize(val) {
-        return my(this).publishingMarshaller.serialize(val);
-      },
-      /** @param {{ body: string, slots: any[] }} data */
-      unserialize(data) {
-        return my(this).publishingMarshaller.unserialize(data);
+        return makeMarshallers(this.state).readonlyMarshaller;
```

Comment:
This returns a fresh marshaller on each call; that would be `makePublishingMarshaller`. The name `getPublishingMarshaller` means that it returns the same object across calls. We discussed was making a new marshaller inside the call to `serialize` and `unserialize`. But to maintain the external API, we should continue to return the facets.

Reply:
looks like this was later changed back